### ISSUE TITLE
Unstoppable animations fail WCAG 2.1 Success Criterion 2.2.2

### ISSUE THREAD

**Issue by anthonyfok** [OpenDRR/riskprofiler#123]
Pierre Dubois (@duboisp) kindly received me during [WET Office hours, Heures de service de la BOEW on 2023-12-19](https://github.com/wet-boew/wet-boew/wiki/WET-Office-hours,-Heures-de-service-de-la-BOEW/2205dec4aa2d006b25c1e405c76ac2b75417d5e4), during which he briefly reviewed https://www.riskprofiler.ca/ and noticed the looping animated logo which the end-user is unable to stop.  This behaviour fails [WCAG 2.1 Success Criterion 2.2.2](https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html):

> Moving content can also be a severe distraction for some people. Certain groups, particularly those with attention deficit disorders, find blinking content distracting, making it difficult for them to concentrate on other parts of the Web page. Five seconds was chosen because it is long enough to get a user's attention, but not so long that a user cannot wait out the distraction if necessary to use the page.

See <https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html> for in-depth discussions and recommended solutions for WCAG 2.1 SC 2.2.2 compliance.

The logo animation is currently coded as follows:

```html
<div id="header-logo-block-1" class="block-type-animation fw-block first-of-type">
  <div id="header-logo-block-1-animation" class="animation" data-anim-loop="true"
    data-anim-in-view="false" data-anim-rewind="false"
    data-anim-path="site/assets/uploads/2021/10/lf20_mmrbfbcv.json">
</div>
```

where <https://www.[riskprofiler.ca/site/assets/uploads/2021/10/lf20_mmrbfbcv.json> is [Lottie file](https://lottiefiles.com/) which plays 241 frames at 24 fps, i.e. a ~10-second animation:

```json
{
    "v": "5.7.1",
    "fr": 24,
    "ip": 0,
    "op": 240,
    "w": 320,
    "h": 320,
    "nm": "logo-red-header",
    "ddd": 0,
    "assets": [
        ...
```

Special thanks to Pierre (@duboisp) for bringing this to my attention!

## Related links

- [LottieFiles: Download Free lightweight animations for website & apps.](https://lottiefiles.com/)
- [Web Accessibility (ADA, WCAG, Section 508) - Lottie in Web - Let's talk Lottie](https://forum.lottiefiles.com/t/web-accessibility-ada-wcag-section-508/911)

### Viewing/editing Lottie animations

- [Lottie Creator: Create Lottie animations with ease - LottieFiles](https://creator.lottiefiles.com/) (though it does not support Trim paths, Matte Masks, and Merge Path features used in lf20_mmrbfbcv.json)
- [Figma](https://www.figma.com/), including the web version, supports the LottieFiles plugin which can import an existing Lottie file and plays it perfectly.

**Comment by chloe2407** [comment]
Within WordPress admin, unchecking the box 'loop playback' seems to stop the animation after it runs through once. This could potentially close this issue.

![image](https://github.com/user-attachments/assets/dfc897f9-644a-442e-9344-12f813741336)
